src/models/launch_1/README.md

```diff
@@ -21,6 +21,10 @@ The System76 Launch is a configurable keyboard with the following specifications
   - PBT plastic
   - Dye sublimated legend
   - XDA profile
+  - Sizes:
+    - 3x 2U keycaps (both space bars and left shift)
+    - 8x 1.5U keycaps (tab, caps lock, left ctrl, right fn, right shift, enter, backspace, and delete)
```

Comment:
We're using the `fn` abbreviation for Function here but not the `Del` abbreviation for Delete?